assets/component-card.css

```diff
@@ -37,6 +37,7 @@
 
 .card .card__inner .card__media {
   overflow: hidden;
+  z-index: 0;
```

Comment:
I know we try to avoid comments in code, but I'm tempted to include a short one on this line that makes it known it's needed for a safari bug. I could see it appearing as though it's not doing anything and getting removed in the future. Not sure how everyone feels about that..

Reply:
It might be a good idea - CSS isn't very self-explanatory.